src/main/java/no/nav/finnkandidatapi/config/ShedlockConfig.java

```diff
@@ -0,0 +1,26 @@
+package no.nav.finnkandidatapi.config;
+
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableScheduling
+@EnableSchedulerLock(defaultLockAtMostFor = "PT59S")
```

Comment:
Hva betyr "PT59S"?

Reply:
https://en.wikipedia.org/wiki/ISO_8601#Durations

P is the duration designator (for period) placed at the start of the duration representation.
Y is the year designator that follows the value for the number of years.
M is the month designator that follows the value for the number of months.
W is the week designator that follows the value for the number of weeks.
D is the day designator that follows the value for the number of days.
T is the time designator that precedes the time components of the representation.

Reply:
Låsen vil vel være låst i maks 59 sekunder her.
Dette er for å hindre at hvis jobben krasjer så skal ikke låsen være låst for all tid.

Reply:
Smart!